docs/content/how-to/disaster-recovery/backup-and-restore-oadp.md

```diff
@@ -1,5 +1,9 @@
 # Disaster recovery on Hosted Control Planes
 
+!!! important
+
+    This procedure differs uses the Hypershift OADP plugin that automates many tasks. Here we assume you have oadp < 1.5 in your management cluster, if that's not the case, please refer to [the OADP-1.5 DR procedure](backup-and-restore-oadp-1-5.md).
```

Comment:
This doesn't make sense grammatically - `This procedure differs uses the Hypershift OADP plugin`. Is it supposed to be `This procedure is different from the Hypershift OADP plugin procedure`?

Reply:
yes, that was a non reviewed copy/paste msg xD. Nice catch man!

Reply:
Humm I think we just need to remove "differs". I just want to remark that the plugin is used here :)